modules/ROOT/pages/manage-cli.adoc

```diff
@@ -0,0 +1,567 @@
+= Connect and manage the {product}
```

Comment:
You don't have to do this in this PR, but I think maybe this should be at least 2 separate pages. At minimum, I think config profiles should be their own page.

Reply:
I both agree and disagree 😆

I agree that this page should eventually get broken up, and that config profiles is clearly the one to move first. I still have to import a bunch of "CLI management" content over from the code repo README. I think the information architecture will be easier to sort out when that happens.

I somewhat disagree (currently) about moving the help and autocomplete. I see those as more integral to the "get started" flow since they're the biggest UX unlocks for new users. But on the other hand, we don't want to overwhelm users by giving them a huge page to scroll. I just want to capitalize on this being the last "required" page before they start plinking away at commands.